src/wallet/bdb.cpp

```diff
@@ -803,3 +803,67 @@ std::string BerkeleyDatabaseVersion()
 {
     return DbEnv::version(nullptr, nullptr, nullptr);
 }
+
+bool BerkeleyBatch::ReadKey(CDataStream& key, CDataStream& value)
+{
+    if (!pdb)
+        return false;
+
+    // Key
+    SafeDbt datKey(key.data(), key.size());
+
+    // Read
+    SafeDbt datValue;
+    int ret = pdb->get(activeTxn, datKey, datValue, 0);
+    if (ret == 0 && datValue.get_data() != nullptr) {
+        value.write((char*)datValue.get_data(), datValue.get_size());
+        return true;
+    }
+    return false;
+}
+
+bool BerkeleyBatch::WriteKey(CDataStream& key, CDataStream& value, bool overwrite)
+{
+    if (!pdb)
```

Comment:
It's a non-trivial move, so might as well add brackets or make it inline.

Reply:
Will change if another push is needed.